sdk/communication/Azure.Communication.JobRouter/samples/ExceptionPolicyCrud.md

```diff
@@ -0,0 +1,162 @@
+# Azure.Communication.JobRouter Samples - Exception Policy CRUD operations (sync)
+
+## Import the namespaces
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_UsingStatements
+using Azure.Communication.JobRouter;
+```
+
+## Create a client
+
+Create a `RouterClient`.
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_CreateClient
+var routerClient = new RouterClient(Environment.GetEnvironmentVariable("AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING"));
+var routerAdministrationClient = new RouterAdministrationClient(Environment.GetEnvironmentVariable("AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING"));
+```
+
+## Create an exception policy
+
+```C# Snippet:Azure_Communication_JobRouter_Tests_Samples_Crud_CreateExceptionPolicy
+var exceptionPolicyId = "my-exception-policy";
+
+// we are going to create 2 rules:
+// 1. `EscalateJobOnQueueOverFlowTrigger`: triggers when queue has more than 10 jobs already en-queued,
```

Comment:
Including markdown formatting in a code snippet is odd.   Not going to hurt anything, but they render as visual noise in a code block.

Reply:
This has been updated. Markdown formatting has been removed from code snippets